### PR TITLE
Add changes for v5.0

### PR DESCRIPTION
**Pull request by KaustubhShamshery**
### Implementation

- [x] **ready for review**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -1756,6 +1757,32 @@ print(chain.active_chain.api.GetTransaction("655107c300e86ee6e819af1cbfce097db15
 }
 ```
 
+*Note: If the transaction had an data field or code field, it will be displayed*
```
Comment (bb111189) [inline source/index.html.md]: can you provide a sample output that has data or code field?

Comment (KaustubhShamshery) [reply source/index.html.md]: it is there, there are two examples

File: source/index.html.md [hunk 2]
```diff
@@ -2348,6 +2376,97 @@ Returns the initialization (immutable) parameters of a given smart contract, rep
...
+curl -d '{
+    "id": "1",
+    "jsonrpc": "2.0",
+    "method": "GetSmartContractSubState",
+    "params": ["fe001824823b12b58708bf24edd94d8b5e1cfcf7","admins",[\"0x9bfec715a6bd658fcb62b0f8cc9bfa2ade71434a\""]]
+}' -H "Content-Type: application/json" -X POST "https://api.zilliqa.com/"
+```
+
+```javascript
+const smartContractState = await zilliqa.blockchain.getSmartContractSubState(
```
Comment (bb111189) [inline source/index.html.md]: const smartContractSubState

Comment (renlulu) [reply source/index.html.md]: [\"0x9bfec715a6bd658fcb62b0f8cc9bfa2ade71434a\""] seems typo here?

Comment (renlulu) [reply source/index.html.md]: and there also some change on `java sdk` side:
```java
public class App {
    public static void main(String[] args) throws IOException {
        HttpProvider client = new HttpProvider("https://api.zilliqa.com");
        String smartContractState = client.getSmartContractSubState("fe001824823b12b58708bf24edd94d8b5e1cfcf7");
        System.out.println(smartContractState);
    }
}
```

**Pull request merged**